okhttp/third_party/okhttp/java/io/grpc/okhttp/internal/Platform.java

```diff
@@ -199,11 +210,18 @@ private static Provider getAppEngineProvider() {
   }
 
   /**
-   * Select from the available security providers in preference order. If a preferred provider
-   * is not found then warn but continue.
+   * Select from the available security providers in preference order. If a preferred provider is
+   * not found then warn but continue.
    */
   private static Provider getAndroidSecurityProvider() {
-    for (String providerClassName : ANDROID_SECURITY_PROVIDERS) {
+    for (String providerName : ANDROID_SECURITY_PROVIDERS) {
```

Comment:
We should loop through the providers in order and choose the first we can use. That way the user has some way of configuring our behavior, and our behavior will better match expectations.

Reply:
Done